dali/ft/filecopy.cpp

```diff
@@ -3211,7 +3211,8 @@ void FileSprayer::updateTargetProperties()
                      ((stricmp(aname,"@blockCompressed")==0)&&copyCompressed) ||
                      ((stricmp(aname,"@rowCompressed")==0)&&copyCompressed)||
                      (stricmp(aname,"@local")==0)||
-                     (stricmp(aname,"@recordCount")==0)
+                     (stricmp(aname,"@recordCount")==0) ||
+                     (stricmp(aname,"@expireDays")==0)
```

Comment:
trivial: not new, but strieq would be cleaner.

Or could define the simple ones in a static array/vector/set and use find.